.gitignore

```diff
@@ -1,2 +1,4 @@
 .idea
-.DS_Store
\ No newline at end of file
+.DS_Store
+*.iml
+.target
```

Comment:
put out folder as well to get your code reviewed

Reply:
ok